src/gen_lockstep.erl

```diff
@@ -419,6 +420,9 @@ authorization(UserPass) ->
         end,
     [<<"Authorization: Basic ">>, Auth, <<"\r\n">>].
 
+client_id() ->
+  [<<"X-Client-ID: ">>, atom_to_list(node()), <<"\r\n">>].
```

Comment:
Will `node()` give us something we can correlate to Ion instance IDs? If not, should we plumb the instance ID through to pass here?